internal/autocert/manager_test.go

```diff
@@ -215,12 +217,12 @@ func TestConfig(t *testing.T) {
 
 	mockACME = newMockACME(ca, srv)
 
-	tmpdir := t.TempDir()
+	tmpdir := filepath.Join(os.TempDir(), uuid.New().String())
```

Comment:
Would you mind adding a comment here to mention the issue with t.TempDir(), maybe linking to https://github.com/pomerium/pomerium/issues/4757?